apps/mobile/src/components/widgets/account/utils/format-balance.ts

```diff
@@ -0,0 +1,11 @@
+// FIXME: This is a temporary solution to format the balance. We need to use the same format as the extension.
```

Comment:
I got AI to fire this out for me. I will migrate the `extension` format balance in https://github.com/leather-io/issues/issues/138